tutorials/tour/_posts/2017-02-13-case-classes.md

```diff
@@ -9,135 +9,49 @@ categories: tour
 num: 11
 next-page: pattern-matching
 previous-page: currying
+prerequisite-knowledge: classes, basics, mutability
 ---
 
-Scala supports the notion of _case classes_. Case classes are just regular classes that are:
-
-* Immutable by default
-* Decomposable through [pattern matching](pattern-matching.html)
-* Compared by structural equality instead of by reference
-* Succinct to instantiate and operate on
-
-Here is an example for a Notification type hierarchy which consists of an abstract super class `Notification` and three concrete Notification types implemented with case classes `Email`, `SMS`, and `VoiceRecording`.
+Case classes are like regular classes with a few key differences which we will go over. Case classes are good for modeling immutable data. In the next step of the tour, we'll see how they are useful in [pattern matching](pattern-matching.html).
 
+## Defining a case class
+A minimal case class requires the keywords `case class`, an identifier, and a parameter list (which may be empty):
 ```tut
-abstract class Notification
-case class Email(sourceEmail: String, title: String, body: String) extends Notification
-case class SMS(sourceNumber: String, message: String) extends Notification
-case class VoiceRecording(contactName: String, link: String) extends Notification
-```
+case class Souvenir()
```

Comment:
I’m not sure this is a good example of case class. We generally use case objects when there are no fields. To me, the purpose of case classes is to aggregate information (in their fields).